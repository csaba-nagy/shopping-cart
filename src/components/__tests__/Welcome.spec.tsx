import { render } from '@testing-library/react'
import Welcome from '../Welcome'

describe('Testing Welcome component', () => {
  it ('should render the component properly', () => {
    const { container, getByText } = render(<Welcome />)

    expect(container).toMatchSnapshot()
    expect(getByText('Welcome')).toBeTruthy()
  })
})
