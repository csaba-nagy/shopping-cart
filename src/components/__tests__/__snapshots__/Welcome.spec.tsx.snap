// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Testing Welcome component > should render the component properly 1`] = `
<div>
  <div>
    Welcome
  </div>
</div>
`;
